xml/System.Reflection/Binder.xml

```diff
@@ -314,7 +314,7 @@
         <param name="type">The new <see langword="Type" /> that <paramref name="value" /> will become.</param>
         <param name="culture">An instance of <see cref="T:System.Globalization.CultureInfo" /> that is used to control the coercion of data types. If <paramref name="culture" /> is <see langword="null" />, the <see cref="T:System.Globalization.CultureInfo" /> for the current thread is used.  
   
- Note   For example, this parameter is necessary to convert a <see langword="String" /> that represents 1000 to a <see langword="Double" /> value, because 1000 is represented differently by different cultures.</param>
+ Note   For example, this parameter is necessary to convert a <see langword="String" /> that represents 1000 to a <see cref="T:System.Double" /> value, because 1000 is represented differently by different cultures.</param>
```

Comment:
```suggestion
 Note   For example, this parameter is necessary to convert a string that represents 1000 to a <see cref="T:System.Double" /> value, because 1000 is represented differently by different cultures.</param>
```